src/App.css

```diff
@@ -7,15 +7,30 @@
   height: 80px;
 }
 
-.App-header {
+.imgLogo {
+  height: 100px;
+}
+.cardWrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+}
```

Comment:
Its a good idea to try and put a space in between these classes to seperate them if you can 😄